Documentation update for page parameters [WTF-2288] (#10472)
mendix/docs: content/en/docs/refguide/modeling/pages/on-click-event.md

#### Page Arguments

> **Page arguments** are automatically configured based on the [parameters](/refguide/page-parameter/) of the selected page and the available arguments. In general, arguments are taken from any enclosing data widget. If the data widget enclosing the widget calling a page is inside another (nested) data widget, then objects from that data widget and any others in which it is nested can also be passed.

Reviewer: The last sentence of this paragraph does not really add any new information to the sentence preceding it.

```suggestion
**Page arguments** are automatically configured based on the [parameters](/refguide/page-parameter/) of the selected page and the available arguments. In general, arguments are taken from any enclosing data widget or the page itself.
```
